cmd/derper/bootstrap_dns.go

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2021 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"expvar"
+	"log"
+	"net"
+	"net/http"
+	"strings"
+	"sync"
+	"time"
+)
+
+var (
+	dnsMu    sync.Mutex
+	dnsCache = map[string][]net.IP{}
+)
+
+var bootstrapDNSRequests = expvar.NewInt("counter_bootstrap_dns_requests")
+
+func refreshBootstrapDNSLoop() {
+	if *bootstrapDNS == "" {
+		return
+	}
+	for {
+		refreshBootstrapDNS()
+		time.Sleep(time.Hour)
```

Comment:
Our TTL on login.tailscale.com right now is 10 minutes. I'm afraid we will forget about this and it will lead to confusion during some production debug session.

Is the TTL of the record easily accessible? Can we use it here?

Reply:
It's not easily accessible on all platforms. Notably on Windows. How about just reducing this to 10 minutes?

Reply:
Sure, that's a fine start.

Reply:
Done.